docs/release-notes/december-2023.md

```diff
@@ -0,0 +1,136 @@
+  does not traverse the entire repository on every scan to determine if the
+  files are valid targets for scanning; this improves scan times.
+* Users can now ignore findings locally in Semgrep IDE extensions. These changes
+  persist between restarts.
+* The metrics collected now include more granular information to help
+differentiate scans using different engine capabilities, such as intraprocedural
+scans without secrets validation versus intraprocedural scans *with* secrets
+validation.
+* **CLI tool**: Added new `semgrep test` subcommand, which is an alias for
+`semgrep scan --test`. **Note**: When running Semgrep on a test directory, use
+`semgrep scan test` to avoid confusion with the new `semgrep test` subcommand.
+
+### Changed
+
+* Fixed issue where empty tables in `pyproject.toml` files would fail to parse.
+* **OCaml**: Switched to a tree-sitter-based parser instead of the Menhir
+  parser.
+* **Rust**: Updated the parser used for Rust.
+
+### Fixed
+
+* Fixed issue where webhooks stopped working.
+* Fixed issue so that clicking **Start Tour** now restarts the Getting Started
+  tutorial.
+* Fixed issue where the **Members** page doesn't display a user's new role until
+  the page reloads. <!--OS-1233-->
+* Fixed issue where users switching organizations would result in a 404.
+  <!--OS-136-->
+* Fixed the **Connect to** button under **Settings** > **Source Code Managers**
+  so that it displays correctly based on whether the user can connect to a
+  source code manager. <!-- https://github.com/semgrep/semgrep-app/pull/11812
+  -->
+* **CLI tool**: Updated CLI error message to clarify that users should log in
+  before running either:
+  * `semgrep ci`
+  * `semgrep scan` then `semgrep scan --config`
+  
+## 💻 Code
+
+### Fixed
+
+* Fixed issue where Semgrep Code findings marked as **fixed** can be triaged via
+  the rule group. If the rule is fixed, the triage status cannot be changed back
+  to **ignored**.
+  <!--FIND-1453-->
+* Fixed issue where the rule information card and the rule preview are missing
+  for older findings; all findings now display this information.
+  <!--FIND-1433-->
+* Fixed issue where the finding's severity displayed doesn't match the rule's
+  severity once the rule has been updated. <!--FIND-1397-->
+
+## 🤖 Assistant (beta)
+
+### Added
+
+* Added the **Analyze** button, which triggers all Assistant functions,
+including autofix, autotriage, and component tagging, on selected findings.
+After Assistant performs these functions, users can see their results if they
+filter for findings based on **Recommendation** or by **Component**.
+Additionally, users that choose **No Grouping** instead of **Group by Rule** see
+false positive and true positive recommendations when viewing their finding
+details pages. 
+
+## 🔐 Secrets (beta)
+
+### Added
+
+* Added support for custom validator rules. Note that custom validator rules are
```

Comment:
How can users make use of this feature, i.e. write their own validator rules? (I believe Sal is the engineer for this.)

Reply:
I think I added what you had in mind